Library/Formula/capnp.rb

```diff
@@ -14,6 +14,7 @@ class Capnp < Formula
   option "without-shared", "Disable building shared library variant"
 
   def install
+    ENV.cxx11 if MacOS.version < :mavericks
```

Comment:
C++11 support is obligatory for this package, so I think it's fine to be explicit here, but I'm not sure whether this will above fix the issue. The "missing" headers are present in the ML bottle, but perhaps they're missing the expected functions? 

In any case, I need to make some adjustments here as upstream has moved to cmake and the `without-shared` has apparently been removed.